src/agent/onefuzz-task/src/tasks/coverage/generic.rs

```diff
@@ -138,9 +139,17 @@ impl CoverageTask {
             return Ok(CmdFilter::default());
         };
 
-        // Ensure users can locate the filter relative to the setup container.
-        let expand = Expand::new().setup_dir(&self.config.common.setup_dir);
-        let filter_path = expand.evaluate_value(raw_filter_path)?;
+        let resolved =
+            resolve_setup_relative_path(&self.config.common.setup_dir, raw_filter_path).await?;
+        let filter_path = if let Some(path) = resolved {
+            path
+        } else {
+            error!(
+                "unable to resolve setup-relative coverage filter path: {}",
+                raw_filter_path
+            );
+            return Ok(CmdFilter::default());
```

Comment:
Note the behavior here: if a `coverage_filter` was specified, but we couldn't find it, fall back to the default (empty) filter.

Alternatively, we may just fail the task. Currently, this is the only way to proactively inform the user that their `coverage_filter` subpath was invalid (present, but not resolvable under `setup_dir`).